docker-compose.yml

```diff
@@ -1,7 +1,6 @@
-version: '3.8'
+version: "3.8"
```

Comment:
Version is a deprecated docker compose keyword now. It can and should be removed.

Reply:
Completed with [7656bdf](https://github.com/cznethub/cznet-discovery/pull/36/commits/7656bdfd0577d70387ae981abdfd4e912cddb98a)